### PR TITLE
fix(docs/refactor-examples): enhance chart documentation

### PR DESCRIPTION
**Pull request by matthiashader**
## 💡 What is the current behavior?
Currently only one preview example exists for the charting topic, which is not suifficient and doesn't show the capabilities.

GitHub Issue Number: #N/A, IX-1551

## 🆕 What is the new behavior?

To show the capabilities of charting with Apache ECharts and the corresponding iX theme, new preview examples for all frameworks have been created.
- line chart
- bar chart
- gauge / circle
- 3d 
- functionalities
- empty state

## 🏁 Checklist

A pull request can only be merged if all of these conditions are met (where applicable):

- [ ] 🦮 Accessibility (a11y) features were implemented
- [ ] 🗺️ Internationalization (i18n) - no hard coded strings
- [ ] 📲 Responsiveness - components handle viewport changes and content overflow gracefully
- [ ] 📄 Documentation was reviewed/updated (`pnpm run docs`)
- [ ] 🧪 Unit tests were added/updated and pass (`pnpm test`)
- [ ] 📸 Visual regression tests were added/updated and pass ([Guide](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md#visual-regression-testing))
- [ ] 🧐 Static code analysis passes (`pnpm lint`)
- [ ] 🏗️ Successful compilation (`pnpm build`, changes pushed)

### CODE REVIEW

File: packages/angular-test-app/src/preview-examples/echarts-bar-horizontal-stacked.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<div
+  echarts
+  [options]="options"
+  [theme]="theme"
+  style="
+    display: block;
+    position: relative;
+    width: 100%;
+    height: 40rem;
+    padding-top: 1rem;
```
Comment (danielleroux) [inline packages/angular-test-app/src/preview-examples/echarts-bar-horizontal-stacked.html:19]: use examples-style project for general styles

File: packages/angular-test-app/src/preview-examples/echarts-empty-state.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    id="empty-state-container"
+    style="
+      position: absolute;
+      justify-content: center;
+      align-content: center;
+      width: 100%;
+      height: 100%;
+      background-color: var(--theme-color-backdrop);
+      z-index: 1;
+    "
```
Comment (danielleroux) [inline packages/angular-test-app/src/preview-examples/echarts-empty-state.html:29]: no inline styles, use examples-styles project

File: packages/angular-test-app/src/preview-examples/echarts-empty-state.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    themeSwitcher.themeChanged.on((theme: string) => {
+      this.theme = convertThemeName(theme);
+    });
+  }
+
+  ngAfterViewInit() {
+    if (this.data.value.length === 0) {
+      const overlay = document.getElementById('empty-state-container');
+      overlay!.style.display = 'block';
+    }
```
Comment (danielleroux) [inline packages/angular-test-app/src/preview-examples/echarts-empty-state.ts:55]: Use *ngIf directive for angular conditional rendering

File: packages/angular-test-app/src/preview-examples/echarts-line-multiple-y-axis.ts [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+@Component({
+  selector: 'app-example',
+  templateUrl: './echarts-line-multiple-y-axis.html',
+})
+export default class EchartsLineMultipleYAxis implements OnInit {
+  theme = convertThemeName(themeSwitcher.getCurrentTheme());
+
+  dates = Array.from({ length: 2025 - 2013 }, (_, i) => (2013 + i).toString());
+
+  //prettier-ignore
```
Comment (danielleroux) [inline packages/angular-test-app/src/preview-examples/echarts-line-multiple-y-axis.ts:30]: We should not ignore this

File: packages/documentation/docs/controls/charts/overview.md [hunk 5]
```diff
@@ -25,6 +28,14 @@ import { registerTheme } from '@siemens/ix-echarts';
 registerTheme(echarts);
 ```
 
+<div className="siemens-brand-section">
```
Comment (danielleroux) [inline packages/documentation/docs/controls/charts/overview.md:31]: What should do the `siemens-brand-section`?

File: packages/echarts/src/computecss.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+export default function getComputedCSSProperty(cssValue: string) {
```
Comment (danielleroux) [inline packages/echarts/src/computecss.ts:11]: Library internal function we should take over the prefixing e.g

```ts
....getPropertyValue(`--theme-color-${cssValue}`);
```

the user can use it then with:

```
color: getComputedCSSProperty('neutral'),
```

**Review by danielleroux — CHANGES_REQUESTED** [review]
lgtm in general we should remove all the inline styles

File: packages/documentation/docs/controls/charts/overview.md (hunk 5 above)
Comment (matthiashader) [reply packages/documentation/docs/controls/charts/overview.md]: removed

File: packages/echarts/src/computecss.ts (hunk 6 above)
Comment (matthiashader) [reply packages/echarts/src/computecss.ts]: I have prefixed it as far as possible ....getPropertyValue(`--theme-${cssValue}`); 
We have --theme-color and --theme-chart in use

File: packages/angular-test-app/src/preview-examples/echarts-bar-horizontal-stacked.ts [hunk 7]
```diff
@@ -16,6 +16,7 @@ import { BarSeriesOption, EChartsOption } from 'echarts';
 @Component({
   selector: 'app-example',
   templateUrl: './echarts-bar-horizontal-stacked.html',
+  styles: [`@import 'example-styles/dist/charts.css'`],
```
Comment (danielleroux) [inline packages/angular-test-app/src/preview-examples/echarts-bar-horizontal-stacked.ts:19]: Dont to inline style to import a css use styleUrl/styleUrls (https://v17.angular.io/api/core/Component#styleUrls)

Counts for every new angular component you created

**Review by danielleroux — CHANGES_REQUESTED** [review]

**Pull request merged**